Index: app/src/main/java/tenrikyo/ofudesaki/kr/QuizActivity.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/tenrikyo/ofudesaki/kr/QuizActivity.kt b/app/src/main/java/tenrikyo/ofudesaki/kr/QuizActivity.kt
--- a/app/src/main/java/tenrikyo/ofudesaki/kr/QuizActivity.kt	(revision 2e8667cada9fbfafeac8201bcb9315a295d11003)
+++ b/app/src/main/java/tenrikyo/ofudesaki/kr/QuizActivity.kt	(date 1776009125804)
@@ -9075,7 +9075,7 @@
                         "それよりでけたにんけんである",
                 english = "kono yo no jii to ten towa jitsu no Oya\n" +
                         "soreyori deketa ningen de aru",
-                commentary = "54. 하늘과 땅은 진실한 어버이인 월덕수기의리와 일덕화기의리가 나타난 것이며, 인간은 어버이신의 수호로 그 사이에서 태어났기 대문에 모두 어버이신의 귀여운 자녀이고 인간끼리는 모두 형제자며이다. \n하늘은 달님인 월덕수기의 리이고, 땅은 해님인 일덕화기의리이다. 태초에 월일 어버이신님은 남자추형종자의리와 여자추형묘상의리의 몸 속에 듭시어 인간을 창조하셨다. 이 월일의 리는 현재 땅과 하늘의 모습으로 나타나 있으며, 인간은 그 사이에서 태어나 살아가고 있다. 즉, 땅과 하늘은 인간의 진실한 어버이이다. 인간은 천지를 포괄하시는 어버이신님의 품속에 안겨서 밤낮없이 베풀어 주시는 어버이신님의 수호로 살아가고 있다. 신악가에서\n 이 세상 땅과 하늘의 본을 받아서 부부를 점지하여 왔었으므로 이것이 이 상의 시초이니라\n 고 하신 말씀은 바로 이러한 뜻이다.
+                commentary = "54. 하늘과 땅은 진실한 어버이인 월덕수기의리와 일덕화기의리가 나타난 것이며, 인간은 어버이신의 수호로 그 사이에서 태어났기 대문에 모두 어버이신의 귀여운 자녀이고 인간끼리는 모두 형제자며이다. \n하늘은 달님인 월덕수기의 리이고, 땅은 해님인 일덕화기의리이다. 태초에 월일 어버이신님은 남자추형종자의리와 여자추형묘상의리의 몸 속에 듭시어 인간을 창조하셨다. 이 월일의 리는 현재 땅과 하늘의 모습으로 나타나 있으며, 인간은 그 사이에서 태어나 살아가고 있다. 즉, 땅과 하늘은 인간의 진실한 어버이이다. 인간은 천지를 포괄하시는 어버이신님의 품속에 안겨서 밤낮없이 베풀어 주시는 어버이신님의 수호로 살아가고 있다. 신악가에서\n 이 세상 땅과 하늘의 본을 받아서 부부를 점지하여 왔었으므로 이것이 이 상의 시초이니라\n 고 하신 말씀은 바로 이러한 뜻이다."
             )
         )
         allContent.add(
